### ISSUE TITLE
Crashed Dump on DropBox

### ISSUE THREAD

**Issue by SvenSiwek** [Duke-Jones/RegulatedNoise#42]
Hi

That is the Link

https://www.dropbox.com/s/zn45dogxm5zdykx/RegulatedNoiseDump.dmp?dl=0

Sometimes before the app crashes I get a missing dll error (msvcp120.dll) .

Better you check if one as that dll on his pc ...

Ahh and the app crashes on Tab "EDDN" -> "Start Listening"

Next I try to find and install that dll

yours
Sven

**Comment by Duke-Jones** [comment]
does it still happen with the 0.21 ?

**Comment by SvenSiwek** [comment]
Hi

I installed the missing dll. Sometimes 0.20 did crash.
A few hours ago I installed 0.21 and it crashed I think while I was in the OCR training part.

I deployed the Dump here https://www.dropbox.com/s/zn45dogxm5zdykx/RegulatedNoiseDump.dmp?dl=0

Let me know if it helps - so I can erase the file if not more needed.

You do a cool great job wih that app. I hope you will fix the issues and sometimes get a 1.0 Version :)
Let me know if sending dump files help you, so I will deploy more on my dropbox.

yours
Sen
